src/api/raw.rs

```diff
@@ -196,12 +196,86 @@ pub struct ListParams {
     pub timeout: Option<u32>
 }
 
-/// Common query parameters for put/post/patch calls
+/// Common query parameters for put/post calls
 #[derive(Default, Clone)]
 pub struct PostParams {
     pub dry_run: bool,
 }
 
+/// Common query parameters for patch calls
+#[derive(Default, Clone)]
+pub struct PatchParams {
+    pub dry_run: bool,
+    /// Strategy which will be used. Defaults to `PatchStrategy::Strategic`
+    pub patch_strategy: PatchStrategy,
+    /// force Apply requests. Applicable only to `PatchStrategy::Apply`
+    pub force: bool,
+    /// fieldManager is a name of the actor that is making changes. Required for `PatchStrategy::Apply`
+    /// optional for everything else
+    pub field_manager: Option<String>
+}
```

Comment:
This looks good. It's a first time we put a non-query param inside a patch param, but it goes in the header directly, so it feels right.